python/astropy.rst

```diff
@@ -28,17 +28,20 @@ For reading and writing files in FITS format, both :mod:`astropy.io.fits` and `f
 These libraries differ in the features available and in performance (with the latter usually surpassing the former).
 Developers must evaluate which works best for their use case and choose one accordingly.
 
+
 .. _fitsio: https://github.com/esheldon/fitsio
 
 .. warning::
 
     ``lsst.afw.fits`` must not be used in any new Python code, as it is not considered memory safe.
 
+If not interacting with C++ classes the use of ``astropy.table`` is encouraged.
```

Comment:
Maybe add: "In particular, the ``Table`` class has beneficial properties over Pandas ``DataFrame``." ?